app/src/main/java/no/ntnu/klubbhuset/ui/managerviews/CreateOrganizationForm.java

```diff
@@ -0,0 +1,125 @@
+package no.ntnu.klubbhuset.ui.managerviews;
+
+import android.content.Context;
+import android.media.Image;
+import android.net.Uri;
+import android.os.Bundle;
+
+import androidx.annotation.Nullable;
+import androidx.fragment.app.Fragment;
+import androidx.lifecycle.ViewModelProviders;
+import androidx.navigation.Navigation;
+
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.TextView;
+
+import java.math.BigDecimal;
+
+import no.ntnu.klubbhuset.R;
+import no.ntnu.klubbhuset.data.model.Club;
+
+
+/**
+ * A simple {@link Fragment} subclass.
+ * Activities that contain this fragment must implement the
+ * {@link CreateOrganizationForm.OnFragmentInteractionListener} interface
+ * to handle interaction events.
+ * Use the {@link CreateOrganizationForm#newInstance} factory method to
+ * create an instance of this fragment.
+ */
+public class CreateOrganizationForm extends Fragment {
+    // TODO: Rename parameter arguments, choose names that match
+    // the fragment initialization parameters, e.g. ARG_ITEM_NUMBER
+    private static final String ARG_PARAM1 = "param1";
+    private static final String ARG_PARAM2 = "param2";
+
+    // TODO: Rename and change types of parameters
+    private String mParam1;
+    private String mParam2;
+
+    private OnFragmentInteractionListener mListener;
+
+    public CreateOrganizationForm() {
+        // Required empty public constructor
+    }
+
+
+    @Override
+    public View onCreateView(LayoutInflater inflater, ViewGroup container,
+                             Bundle savedInstanceState) {
+        // Inflate the layout for this fragment
+        return inflater.inflate(R.layout.fragment_create_new_organization, container, false);
+    }
+
+    @Override
+    public void onActivityCreated(@Nullable Bundle savedInstanceState) {
+        super.onActivityCreated(savedInstanceState);
+        Button registerBtn = getView().findViewById(R.id.register_organization);
+        Button cancelBtn = getView().findViewById(R.id.cancel_registration);
+
+        registerBtn.setOnClickListener(l -> onCreateButtonPressed());
+        cancelBtn.setOnClickListener(l -> onCancelButtonPressed());
+    }
```

Comment:
I think this could be done in `onCreateView()`?

Reply:
Løst https://github.com/0us/IE303911-Klubbhuset/commit/f5f01256e6ba81bd07541be7bd201d3c859b2c87